src/testing/fuzz.rs

```diff
@@ -0,0 +1,299 @@
+pub use crate::testing::mock::NetworkData;
+use crate::{
+    nodes::NodeIndex,
+    testing::mock::{
+        configure_network, spawn_honest_member, Data, Hasher64, NetworkHook, PartialMultisignature,
+        Signature, Spawner,
+    },
+    Network, NetworkData as ND, SpawnHandle,
+};
+use codec::{Decode, Encode, IoReader};
+use futures::{
+    channel::oneshot::{self, Receiver},
+    StreamExt,
+};
+use futures_timer::Delay;
+use log::{error, info};
+use std::{
+    io::{BufRead, BufReader, Read, Result as IOResult, Write},
+    iter::once_with,
+    time::Duration,
+};
+use tokio::runtime::{Builder, Runtime};
```

Comment:
Still `tokio` stuff, but I totally don't understand why it's here – are you completely sure it cannot be removed?

Reply:
mock.rs still uses tokio::spawn and so I need to provide it some tokio::Runtime. I've checked futures::Executor::block_on and it was panicing. Or perhaps I can just remove those tokio things from mock.rs instead.